scripts/deploy-demo.ts

```diff
@@ -0,0 +1,20 @@
+import { join } from 'path';
+import { sh } from './utils';
+import { execSync } from 'child_process';
+
+function shVerbose(command: string) {
+  execSync(command, { stdio: 'inherit' });
+}
+
+// Get the name of the branch we're on
+const branch = sh('git rev-parse --abbrev-ref HEAD');
+const urlPath = `/design-system/branch/${branch}`;
+
+// Build the demo site
+shVerbose(`yarn --cwd ${join('packages', 'docs')} clean`);
+shVerbose(`PATH_PREFIX='${urlPath}' PREFIX_PATHS=true yarn build:docs`);
```

Comment:
Just so I understand what's going on here:
1. We create a urlPath var that contains our branch name
2. We manually override our PATH_PREFIX env var before building our docs. 

Is that correct?

Reply:
Yep, that's correct